public_desktop/desktop-app.js

```diff
@@ -180,6 +180,16 @@ function startDesktopApp() {
 	}
 }
 
+// hard code wrongly interpreted words in spellchecker
+// electron has a bug with parsing of contractions
+// Ref: https://github.com/atom/electron/issues/1005
+function skipWords( locale, text ) {
+	// locale == 'en-us' already skipped in setup
+	var words = [ 'ain', 'couldn', 'didn', 'doesn', 'hadn', 'hasn', 'mightn', 'mustn', 'needn', 'oughtn', 'shan', 'shouldn', 'wasn', 'weren', 'wouldn'];
```

Comment:
Also, the `in` check doesn't work here. I think we need an `indexOf` instead: `return words.indexOf( text ) !== -1;` instead.